.gitignore

```diff
@@ -1,3 +1,4 @@
 private*
 values-private.yaml
 charts/
+/*.tgz
```

Comment:
Why is that change required? I don't see any changes that requires the manipulation of tarballs.

if it's because of files you had in your clone while developing/troubleshooting that you didn't want to commit by mistake, I propose to ignore `tmp/` and add that dir to `.gitignore`.

Reply:
That's a common extension to ignore on a Helm repository, whenever you run `helm template ...` it creates the tarball -- which should always be ignored.